### PR TITLE
Use official tracing package 📦

### LINKED ISSUE #3: How to transform array attribute into Zipkin V2 tag
@ktoso [brought up a valid concern](https://github.com/slashmo/jaeger-client-swift/pull/1#discussion_r502349787) about transforming an `.array` attribute into a Zipkin tag in case the array elements are strings:

> what if the string is "hello, oh no" -> [foo, hello, ohno]

I'm not quite sure how to handle this, though. Let's leave this issue as a reminder for now.

### PR DESCRIPTION
**Pull request by slashmo**
Now that Tracing has been officially released, I switched from [slashmo/gsoc-swift-tracing](https://github.com/slashmo/gsoc-swift-tracing) to [apple/swift-distributed-tracing](https://github.com/apple/swift-distributed-tracing). 

This involved some minor refactoring and rewording, and as a positive side-effect closed #3 using the new `SpanAttribute` type-specific arrays like `SpanAttribute.intArray`.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -8,22 +8,19 @@ let package = Package(
...
-        .package(url: "https://github.com/slashmo/gsoc-swift-tracing.git", .branch("main")),
+        .package(url: "https://github.com/apple/swift-distributed-tracing.git", from: "0.1.1"),
         .package(url: "https://github.com/slashmo/swift-w3c-trace-context.git", from: "0.6.0"),
-        .package(url: "https://github.com/slashmo/swift-nio.git", .branch("feature/baggage-context")),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
     ],
     targets: [
         .target(name: "Jaeger", dependencies: [
-            .product(name: "Tracing", package: "gsoc-swift-tracing"),
+            .product(name: "Tracing", package: "swift-distributed-tracing"),
```
Comment (ktoso) [inline Package.swift:17]: excellent :)

File: Sources/Jaeger/JaegerSpan.swift [hunk 2]
```diff
@@ -23,9 +23,9 @@ public final class JaegerSpan: Span {
         self.baggage.traceContext?.sampled ?? false
     }
 
-    public let startTimestamp: Timestamp
+    public let startTimestamp: DispatchWallTime
```
Comment (ktoso) [inline Sources/Jaeger/JaegerSpan.swift:26]: 👍

**Review by ktoso — APPROVED** [review]
Looking good :)

**Comment by slashmo** [comment]
Thanks for the quick review, @ktoso 🚀

**Pull request merged**